### PR TITLE
 added SelectRootOnlySwitch

### PR DESCRIPTION
**Pull request by sahandfarahani**

### CODE REVIEW

File: src/components/MarkdownDisplay.tsx [hunk 1]
```diff
@@ -15,11 +17,24 @@ interface Props {
...
+  const [filter, setFilter] = useState<string | null>(null);
+  const filtering = (filter): Function | null => {
+    if (filter === "ROOT_ONLY") {
+      return selectRootCores;
+    } else {
+      return null;
+    }
+  };
+
+  useLayoutEffect(() => {
```
Comment (ralph-dev) [inline src/components/MarkdownDisplay.tsx:29]: Why not useEffect??

Comment (sahandfarahani) [reply src/components/MarkdownDisplay.tsx:29]: Yes... I was trying something different, I forgot to put it back to useEffect

**Comment by ralph-dev** [comment]
You are good to merge if you don't see an application for Typescript enums anywhere :)

**Comment by sahandfarahani** [comment]
> You are good to merge if you don't see an application for Typescript enums anywhere :)

I tried to push but again I had to pull and then it didn't get pushed
so I'll merge this one and make a PR for enum again

**Pull request merged**